src/main/java/nvt/kts/ticketapp/domain/dto/event/EventDTO.java

```diff
@@ -0,0 +1,24 @@
+package nvt.kts.ticketapp.domain.dto.event;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import nvt.kts.ticketapp.domain.model.event.EventCategory;
+
+@NoArgsConstructor
```

Comment:
Mogli smo samo da rokamo @Data gore, ne treba ni konstruktor s parametrima sve napravi.

Reply:
Moze, samo sto @Data ne podrzava @NoArgsConstructor.
Iz dokumentacije: "@Data: All together now: A shortcut for @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor!"

Reply:
Ali necemo sad menjati bezveze, al ubuduce cemo koristiti